website/docs/d/config_aggregate_compliance_packs.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+subcategory: "Cloud Config"
+layout: "alicloud"
+page_title: "Alicloud: alicloud_config_aggregate_compliance_packs"
+sidebar_current: "docs-alicloud-datasource-config-aggregate-compliance-packs"
+description: |-
+  Provides a list of Config Aggregate Compliance Packs to the user.
+---
+
+# alicloud\_config\_aggregate\_compliance\_packs
+
+This data source provides the Config Aggregate Compliance Packs of the current Alibaba Cloud user.
+
+-> **NOTE:** Available in v1.124.0+.
+
+## Example Usage
+
+Basic Usage
+
+```terraform
+data "alicloud_config_aggregate_compliance_packs" "example" {
+  aggregator_id = "ca-3a9b626622af001d****"
+  ids           = ["ca-3a9b626622af001d****:cp-152a626622af00bc****]
```

Comment:
The value should be a list of the sub resource ids.

Reply:
ok